examples/tensorflow/mnist_utils.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2024 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Is this Python script just copy and paste from TF community?

Reply:
This is a script I wrote, not copy & paste. 
My mistake, I meant to write a copyright for **kubeflow** because it's a new document, but do we really need this copyright? (if so, I'll fix the mistake and change it to kubeflow, otherwise I'll delete it)

Reply:
I see. In that case, could we use the kubeflow copyright?

Reply:
Done